src/index.html

```diff
@@ -11,12 +11,72 @@
       content="ie=edge"
     />
     <title>Moyo header</title>
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="container">
+        <div class="logo_container">
+          <a
+            class="logo_link"
+            href="index.html"
+          >
+            <img
+              src="./images/logo.png"
+              alt="Logo"
+            />
+          </a>
+        </div>
+        <div class="nav_container">
+          <ul>
+            <li>
+              <a
+                href="#"
+                class="is-active"
+              >
+                Apple
+              </a>
+            </li>
+            <li>
```

Comment:
```suggestion
            </li>
            
            <li>
```
The same in all places

<img width="1038" alt="image" src="https://github.com/mate-academy/layout_moyo-header/assets/86789161/4dbed4fc-d223-44d5-b2fc-a4a86a03916e">

Reply:
Got it